src/coreComponents/physicsSolvers/inducedSeismicity/CMakeLists.txt

```diff
@@ -1,27 +1,61 @@
+# Copyright (c) 2023-2024 Chevron
+# Copyright (c) 2019-     GEOS/GEOSX Contributors
+# All rights reserved
+#
+# See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+#
+#--------------------------------------------------------------------------------------------------
+
+#[[
+Package: inducedSeismicitySolvers
+Contains:
+#]]
+
 # Specify solver headers
-set( physicsSolvers_headers
-     ${physicsSolvers_headers}
-     inducedSeismicity/ExplicitQDRateAndState.hpp
-     inducedSeismicity/ImplicitQDRateAndState.hpp
-     inducedSeismicity/inducedSeismicityFields.hpp
-     inducedSeismicity/QDRateAndStateBase.hpp
-     inducedSeismicity/QuasiDynamicEarthQuake.hpp
-     inducedSeismicity/rateAndStateFields.hpp
-     inducedSeismicity/SeismicityRate.hpp
-     inducedSeismicity/SpringSlider.hpp
-     inducedSeismicity/kernels/ExplicitRateAndStateKernels.hpp
-     inducedSeismicity/kernels/ImplicitRateAndStateKernels.hpp
-     inducedSeismicity/kernels/RateAndStateKernelsBase.hpp
-     inducedSeismicity/kernels/SeismicityRateKernels.hpp
-     PARENT_SCOPE )
+set( inducedSeismicitySolvers_headers
+     ExplicitQDRateAndState.hpp
+     ImplicitQDRateAndState.hpp
+     inducedSeismicityFields.hpp
+     QDRateAndStateBase.hpp
+     QuasiDynamicEarthQuake.hpp
+     rateAndStateFields.hpp
+     SeismicityRate.hpp
+     SpringSlider.hpp
+     kernels/ExplicitRateAndStateKernels.hpp
+     kernels/ImplicitRateAndStateKernels.hpp
+     kernels/RateAndStateKernelsBase.hpp
+     kernels/SeismicityRateKernels.hpp )
 
 # Specify solver sources
-set( physicsSolvers_sources
-     ${physicsSolvers_sources}
-     inducedSeismicity/ExplicitQDRateAndState.cpp 
-     inducedSeismicity/ImplicitQDRateAndState.cpp 
-     inducedSeismicity/QDRateAndStateBase.cpp
-     inducedSeismicity/QuasiDynamicEarthQuake.cpp
-     inducedSeismicity/SeismicityRate.cpp
-     inducedSeismicity/SpringSlider.cpp
-     PARENT_SCOPE )
+set( inducedSeismicitySolvers_sources
+     ExplicitQDRateAndState.cpp 
+     ImplicitQDRateAndState.cpp 
+     QDRateAndStateBase.cpp
+     QuasiDynamicEarthQuake.cpp
+     SeismicityRate.cpp
+     SpringSlider.cpp )
+
+set( dependencyList ${parallelDeps} physicsSolversBase )
+
+geos_decorate_link_dependencies( LIST decoratedDependencies DEPENDENCIES ${dependencyList} )
+
+blt_add_library( NAME       inducedSeismicitySolvers
+                 SOURCES    ${inducedSeismicitySolvers_sources}
+                 HEADERS    ${inducedSeismicitySolvers_headers}
+                 DEPENDS_ON ${decoratedDependencies} ${externalComponentDeps}
+                 OBJECT     ${GEOS_BUILD_OBJ_LIBS}
+                 SHARED     ${GEOS_BUILD_SHARED_LIBS}
+               )
+
+target_include_directories( inducedSeismicitySolvers PUBLIC ${CMAKE_SOURCE_DIR}/coreComponents )
+
+install( TARGETS inducedSeismicitySolvers LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib )
```

Comment:
@white238 is this install command always necessary after adding a library with BLT? Or will blt  add it to the intall dir?

Reply:
BLT does not handle installing. You must call the CMake `install` call yourself. I did at one point, a long time ago, try to come up with a default install strategy but noone could agree on anything and the CMake ones are pretty nice.

Reply:
okay, thank you!